test/bddtests/features/file.feature

```diff
@@ -36,12 +36,16 @@ Feature: Upload files to DCAS which are backed by a Sidetree file index document
     And fabric-cli is executed with args "ledgerconfig update --configfile ./fixtures/config/fabric/org1-config.json --noprompt"
     And fabric-cli is executed with args "ledgerconfig update --configfile ./fixtures/config/fabric/org2-config.json --noprompt"
 
+    Given variable "token_content_w" is assigned the value "TOKEN_CONTENT_W"
```

Comment:
do you have a test for `token_content_r`?

Reply:
This test in in sidetree-fabric - I didn't think it necessary to complicate the fabric-cli test.

Reply:
In sidetree-fabric I defined an /internal path that requires both read and write access.